### PR TITLE
BT-1002 add experience test case

### PR DESCRIPTION
**Pull request by Muradil-Erkin**
- automated the add experience flow
- added happy path and un-happy path

### CODE REVIEW

File: src/test/java/selenium/AddExperienceTest.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package selenium;
+
+public class AddExperienceTest {
+
+	public static void main(String[] args) {
+		
+		System.out.println("Pretending that I automated a test case here");
```
Comment (Muradil-Erkin) [inline src/test/java/selenium/AddExperienceTest.java:7]: Dude I don't like this code, you can do better.

Comment (laloclau) [inline src/test/java/selenium/AddExperienceTest.java:7]: Nice code!

Comment (CarterCChen) [inline src/test/java/selenium/AddExperienceTest.java:7]: sorry, but this is how my teacher taught me.lol

**Review by laloclau — APPROVED** [review]

File: src/test/java/selenium/AddExperienceTest.java (hunk 1 above)
Comment (huidong198706) [inline src/test/java/selenium/AddExperienceTest.java:7]: good job. keep the good work.

**Review by ajf876 — APPROVED** [review]
Looks good.

File: src/test/java/selenium/AddExperienceTest.java (hunk 1 above)
Comment (yadikaer) [inline src/test/java/selenium/AddExperienceTest.java:7]: L code.

**Review by yadikaer — CHANGES_REQUESTED** [review]

**Review by qingyuanre1 — APPROVED** [review]

**Review by EvaGao88 — APPROVED** [review]
good

File: src/test/java/selenium/AddExperienceTest.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
```
Comment (AnnaZhao0209) [inline src/test/java/selenium/AddExperienceTest.java:8]: I am happy to join this team.

File: src/test/java/selenium/AddExperienceTest.java (hunk 1 above)
Comment (minyizeng) [inline src/test/java/selenium/AddExperienceTest.java:7]: code is too easy

**Review by LixiaSun — APPROVED** [review]
Reviewed Completed.

**Review by Habtamu-Ayele — APPROVED** [review]

**Review by tylee17 — COMMENTED** [review]
good

**Review by huidong198706 — APPROVED** [review]

File: src/test/java/selenium/AddExperienceTest.java (hunk 1 above)
Comment (Muradil-Erkin) [reply src/test/java/selenium/AddExperienceTest.java:7]: Im not gonna do it, live with it.

**Pull request merged**